bundles/org.openhab.binding.tapocontrol/src/main/java/org/openhab/binding/tapocontrol/internal/devices/dto/TapoLightEffect.java

```diff
@@ -0,0 +1,229 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.tapocontrol.internal.devices.dto;
+
+import static org.openhab.binding.tapocontrol.internal.TapoControlHandlerFactory.GSON;
+import static org.openhab.binding.tapocontrol.internal.constants.TapoComConstants.*;
+
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.binding.tapocontrol.internal.constants.TapoErrorCode;
+import org.openhab.binding.tapocontrol.internal.helpers.TapoErrorHandler;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Tapo-LightEffects Structure Class
+ *
+ * @author Christian Wild - Initial contribution
+ */
+@NonNullByDefault
+public class TapoLightEffect {
+    @Expose
+    private String id = "";
+
+    @Expose
+    private String name = "";
+
+    @Expose
+    private Integer brightness = 0;
```

Comment:
`int` instead of `Integer` ?

Reply:
Integer is used because @Nullable is necesarry to send null in request for function of the "set request".